packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -521,7 +519,14 @@ class FileCollection extends DocumentCollection {
    */
   async updateFile(
     data,
-    { executable = false, fileId, name = '', metadata, ...options } = {}
+    {
```

Comment:
let's update the above jsdoc 🙏

Reply:
I updated the `FileAttributes` with the new `encrypted` file, did I miss something?

Reply:
yes. I suggest to add the parameter inside the jsdoc line 515

Reply:
Hm, sorry I don't get it. Line 515, we have the [FileAttributes](https://github.com/cozy/cozy-client/blob/24b23dabe8b78fa39132281aa52d36991f8178aa/packages/cozy-stack-client/src/FileCollection.js#L515), that includes the description of the `encrypted` field: https://github.com/cozy/cozy-client/blob/24b23dabe8b78fa39132281aa52d36991f8178aa/packages/cozy-stack-client/src/FileCollection.js#L40
Hence, it is already documented, right? https://github.com/cozy/cozy-client/blob/4c1c3089b65987d4a4697cfeab8cd1ca6ba9e2b5/docs/api/cozy-stack-client.md#fileattributes--object

Reply:
all right, oops 😅